SQLPractice/src/main/java/com/sql/topic5/DBManager.java

```diff
@@ -0,0 +1,115 @@
+package com.sql.topic5;
+
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.FileReader;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Date;
+
+public class DBManager {
+	private static Connection connection;
+	private static final String newLine = System.getProperty("line.separator");
+	public void connect() throws ClassNotFoundException, SQLException {
+		String driverName = "org.gjt.mm.mysql.Driver";
+	    Class.forName(driverName);
+
+	    String serverName = "localhost";
+	    String mydatabase = "";
+	    String url = "jdbc:mysql://" + serverName + "/" + mydatabase; 
+
+	    String username = "root";
+	    String password = "";
+	    connection = DriverManager.getConnection(url, username, password);
+	}
+	
+	public void closeConnection() throws SQLException {
```

Comment:
High: if you close the connection, the singleton is retrieving a closed connection.